notebooks/official/automl/sdk_automl_video_object_tracking_batch.ipynb

```diff
@@ -77,13 +82,14 @@
         "\n",
         "In this tutorial, you learn how to create an AutoML video object tracking model from a Python script, and then do a batch prediction using the Vertex AI SDK. You can alternatively create and deploy models using the `gcloud` command-line tool or online using the Cloud Console.\n",
```

Comment:
Shouldn't this be: "...do a batch prediction using the Vertex AI SDK for Python" ??